nemo_skills/pipeline/utils/exp.py

```diff
@@ -620,6 +629,16 @@ def run_exp(exp, cluster_config, sequential=False, dry_run=False):
     if dry_run:
         LOG.info("Dry run mode is enabled, not running the experiment.")
         return
+
+    if "mounts" in cluster_config:
+        # Can only check cluster mounts here, not those added to add_task
+        mounts = get_mounts_from_config(cluster_config)
+        mount_sources = [m.split(":")[0] for m in mounts]
+
+        LOG.info("Checking mount paths: %s", mount_sources)
+        exit_if_failure = not cluster_config.get("NEMO_SKILLS_DISABLE_MOUNT_CHECK", False)
```

Comment:
was this meant to be an env var? Otherwise, would be more standard and clear to call it `disable_mount_check` (lowercase and without NEMO_SKILLS as it's already config from nemo skills config)